packages/nimble-components/src/anchor/tests/anchor-matrix.stories.ts

```diff
@@ -0,0 +1,78 @@
+import type { Meta, Story } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { pascalCase } from '@microsoft/fast-web-utilities';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../utilities/tests/matrix';
+import {
+    createMatrixThemeStory,
+    createStory
+} from '../../utilities/tests/storybook';
+import { hiddenWrapper } from '../../utilities/tests/hidden';
+import { textCustomizationWrapper } from '../../utilities/tests/text-customization';
+import '../../all-components';
+import { AnchorAppearance } from '../types';
+
+const metadata: Meta = {
+    title: 'Tests/Anchor',
+    decorators: [withXD],
+    parameters: {
+        ...sharedMatrixParameters(),
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46'
```

Comment:
The matrix artboards are still not working for me (though the non-matrix ones are now, thanks!).

Reply:
Whoops. Fixed.